docs/translation.md

```diff
@@ -0,0 +1,49 @@
+## How to add new language translation to Remark42
+
+Translation files are stored in [/frontend/app/locales](https://github.com/umputun/remark42/tree/master/frontend/app/locales)
+directory with `.json` extension and content like following:
+
+```json
+{
+  "anonymousLoginForm.length-limit": "Username must be at least 3 characters long",
+  "anonymousLoginForm.log-in": "Log in",
+  "anonymousLoginForm.symbol-limit": "Username must start from the letter and contain only latin letters, numbers, underscores, and spaces",
+<...>
+}
+```
+
+### How to add a new translation
+
+We truly appreciate people spending time contributing their translations to remark42. Please go through the steps
+below in order to have your translation start being available to all remark42 users and included in the next release.
+
+1. create a fork of [umputun/remark42](https://github.com/umputun/remark42) repo, and if you already have one please
+pull the latest changes from the upstream master branch. It could be done like that:
+   ```shell
+   git remote add upstream https://github.com/umputun/remark42.git
+   git fetch upstream
+   git rebase upstream/master
+   git push
+   ```
+1. copy [frontend/app/locales/en.json](https://github.com/umputun/remark42/blob/master/frontend/app/locales/en.json)
```

Comment:
https://github.com/umputun/remark42/blob/master/frontend/Readme.md#how-to-add-new-locale

Reply:
Thank you, could you please check the updated version?